app-backend/batch/src/main/scala/com/azavea/rf/batch/export/CheckExportStatus.scala

```diff
@@ -150,20 +150,24 @@ case class CheckExportStatus(exportId: UUID, statusURI: URI, time: Duration = 60
       sql"update exports set export_status = ${exportStatus} where id = ${exportId}"
         .update.withUniqueGeneratedKeys("export_status")
 
-    val withLoggingUpdateExportStatus: ConnectionIO[Unit] =
-      updateIo(exportId, s3ExportStatus.exportStatus) map {
-        case ExportStatus.Failed => {
-          logger.info(s"Export finished with ${s3ExportStatus.exportStatus}")
-          sendError(s"Export status update failed for ${exportId}")
-          notifyExportOwner("FAILED")
+    val withLoggingUpdateExportStatus: ConnectionIO[ExportStatus] =
+      updateIo(exportId, s3ExportStatus.exportStatus) <* (
```

Comment:
What's this `<*` thing doing?

Reply:
It takes two monads, performs the side-effects from both, and returns the first one